Cangjie5.txt

```diff
@@ -58867,6 +58864,7 @@
 䧑	nljmo
 院	nljmu
 𨺋	nljnu
+𨺶	nljoe
```

Comment:
It was `mnjoe` before this PR, @Jackchows please double check if `nljoe` is correct, thanks!